### PR TITLE
Update readme to conform to doc standards, and include project governance info

### PR DESCRIPTION
**Pull request by mthaddon**
Update home page documentation per new standards.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,71 @@
...
-your Discourse instance.
+Once the deployment is completed and the "discourse-k8s" workload state in
+`juju status` has changed to "active" you can add `discourse-k8s` to
+`/etc/hosts` with the IP address of your Kubernetes cluster's ingress
+(127.0.0.1 if you're using MicroK8s) and visit `http://discourse-k8s` in a
+browser and log in to your Discourse instance.
+
+## Project and community
+
+The Nginx Ingress Integrator Operator is a member of the Ubuntu family. It's an
```
Comment (arturo-seijas) [inline README.md:58]: Wrong charm name here

**Review by arturo-seijas — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mthaddon) [reply README.md]: Doh! Fixed.

**Review by arturo-seijas — DISMISSED** [review]
LGTM

**Review by merkata — DISMISSED** [review]
Nicely done!

File: README.md [hunk 2]
```diff
@@ -1,37 +1,71 @@
...
+A juju charm deploying and managing Discourse on Kubernetes. Discourse is the
+100% open source discussion platform. Use it as a mailing list, discussion
+forum or long-form chat room.
 
-## Overview
+This charm simplifies initial deployment and "day N" operations of Discourse
+on Kubernetes, such as scaling the number of instances, integration with SSO,
+access to S3 for redundant file storage and more. It allows for deployment on
+many different Kubernetes platforms, from [MicroK8s](https://microk8s.io) to
+[Charmed Kubernetes](https://ubuntu.com/kubernetes) to public cloud Kubernetes
```
Comment (gregory-schiano) [inline README.md:11]: A very small remark:
I'd rather write: "from MicroK8S or Charmed Kubernetes to public cloud..."
Any way to avoid repeating "to"

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Pull request merged**